cloud/availability.rst

```diff
@@ -1,7 +1,7 @@
 .. _ha:
 
-Availability and Recovery
-#########################
+Backup, Availability, and Replication
```

Comment:
Any reason to change Recovery to Replication?

Reply:
Just to catch that search keyword, which is a common one.